examples/local/scripts/vttablet-up.sh

```diff
@@ -47,8 +47,7 @@ vttablet \
  -health_check_interval 5s \
  -enable_semi_sync \
  -enable_replication_reporter \
- -heartbeat_enable \
- -heartbeat_interval 250ms \
+ -heartbeat_enable=false \
```

Comment:
`master` branch already has these removed. seems to me like an update from `master` is useful.